client/src/actions/actions.ts

```diff
@@ -39,7 +39,7 @@ interface RequestsAndUsers {
 export type RequestsAndUsersAction = RequestsAndUsers & GenericAction;
 
 interface RequestStatus {
-    requestId: string;
+    requestId: number;
```

Comment:
why string to number?

Reply:
It was original string because request IDs were once thought to be UUIDs.  But it's clear now they'll be a normal incrementing ID count